src/components/radio/radio.spec.ts

```diff
@@ -1,370 +1,398 @@
 import {
-  it,
-  describe,
-  expect,
-  beforeEach,
-  fakeAsync,
-  inject,
-  tick,
+    it,
+    describe,
+    beforeEach,
+    beforeEachProviders,
+    inject,
+    async,
+    fakeAsync,
+    tick
 } from '@angular/core/testing';
-import {TestComponentBuilder} from '@angular/compiler/testing';
-import {Component, DebugElement} from '@angular/core';
+import {FORM_DIRECTIVES, NgControl} from '@angular/common';
+import {TestComponentBuilder, ComponentFixture} from '@angular/compiler/testing';
+import {Component, DebugElement, provide} from '@angular/core';
 import {By} from '@angular/platform-browser';
-
-import {MdRadioButton, MdRadioGroup, MdRadioChange} from './radio';
+import {PromiseCompleter} from '../../core/async/promise-completer';
+import {MD_RADIO_DIRECTIVES, MdRadioGroup, MdRadioButton} from './radio';
 import {MdRadioDispatcher} from './radio_dispatcher';
 
-export function main() {
-  describe('MdRadioButton', () => {
-    let builder: TestComponentBuilder;
 
-    beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
-      builder = tcb;
+describe('MdRadio', () => {
+  let builder: TestComponentBuilder;
+  let dispatcher: MdRadioDispatcher;
+
+  beforeEachProviders(() => [
+    provide(MdRadioDispatcher, {useFactory: () => {
+      dispatcher = new MdRadioDispatcher();
+      return dispatcher;
+    }})
+  ]);
+
+  beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
+    builder = tcb;
+  }));
+
+  describe('inside of a group', () => {
+    let fixture: ComponentFixture<RadiosInsideRadioGroup>;
+    let groupDebugElement: DebugElement;
+    let groupNativeElement: HTMLElement;
+    let radioDebugElements: DebugElement[];
+    let radioNativeElements: HTMLElement[];
+    let groupInstance: MdRadioGroup;
+    let radioInstances: MdRadioButton[];
+    let testComponent: RadiosInsideRadioGroup;
+
+    beforeEach(async(() => {
+      builder.createAsync(RadiosInsideRadioGroup).then(f => {
+        fixture = f;
+        fixture.detectChanges();
+
+        testComponent = fixture.debugElement.componentInstance;
+
+        groupDebugElement = fixture.debugElement.query(By.directive(MdRadioGroup));
+        groupNativeElement = groupDebugElement.nativeElement;
+        groupInstance = groupDebugElement.injector.get(MdRadioGroup);
```

Comment:
Nope; radio-group is a `@Directive` rather than a `@Component`, so it doesn't have a `componentInstance`.